README.md

```diff
@@ -124,11 +128,28 @@ target algorithms.
 
 # Examples
 
-See examples/
-
-  * examples/rosenbrock.py - example on how to optimize a Python function
-  * examples/spear_qcp/run.sh - example on how to optimize the SAT solver Spear
-    on a set of SAT formulas
+We provide a bunch of examples in the [examples folder](examples), such as:
+
+  * Optimization of a python function directly with SMAC
```

Comment:
Agreed that list could get outdated but it might be nice to keep it up to date.
The list before that was heavily outdated ;) We could also remove the list if you prefer.
As stated above I created the colab example myself.

Reply:
Agreed, the list is a definitive improvement and we should keep it.

How about linking to the [rendered examples in the docs](https://automl.github.io/SMAC3/master/examples/fmin_rosenbrock.html#sphx-glr-examples-fmin-rosenbrock-py), though?

Reply:
Agreed. I changed every link to the rendered examples.
Only the Branin example is not listed there so I kept the link to the example code in the repository and the example of running SMAC for AC of a SAT solver is also not in the rendered examples.